exo-clone/src/cli.ls

```diff
@@ -13,20 +13,11 @@ clone = ->
 
   [_, _, repo-origin] = process.argv
   return missing-origin! unless repo-origin
-  repo = repo-info repo-origin
+  repository = repo-info repo-origin
```

Comment:
The `repository` referred to here is the object that is passed into the `AppCloner`